docs/modules/development/pages/rest/config_management.adoc

```diff
@@ -48,9 +66,27 @@ Configurations modified will take effect instantly.
 | `/cm/\{configName}/\{configId}`             |  Update configuration.
                                                      If configuration does not exist, it will return error.
 
-200: Return nothing
+200: Return: nothing
 
 400: Return: {"MESSAGE": "<ERROR_MESSAGE>"}
+| `/cm/\{configName}/\{configId}/\{path}` |
+Updates a part of configuration specified by configName, configId and the path to the part in JsonPath format. Returns 404 when the configuration does not exist. Returns 400 When the path parameter has a wrong format or specifies a nonexistent path or provided part of configuration has a wrong format.
+
+200 Return: nothing
+
+404 Return: {"MESSAGE": "<ERROR_MESSAGE>"}
+
+400 Return: {"MESSAGE": "<ERROR_MESSAGE>"}
+
+| `/cm/\{configName}/\{configId}/\{pathToParent}/\{nodeName}` |
+Updates or inserts a part of configuration specified by configName, configId, path to the parent node in JsonPath format and node name. Returns 404 when the configuration does not exist. Returns 400 When the pathToParent parameter has a wrong format or specifies a nonexistent path or provided part of configuration has a wrong format.
```

Comment:
```suggestion
Updates or inserts a part of the configuration specified by `configName`, `configId`, and the path to the parent node in JSONPath format with the node name.
Returns 404 when the configuration does not exist.
Returns 400 when the `pathToParent` parameter has a wrong format, specifies a nonexistent path, or the provided part of the configuration has a wrong format.
```
In the first sentence, is the path to the parent node in JSONPath format AND includes the node name? If so, please use my suggestion of "with".

Earlier examples do not specify the `pathToParent` parameter. Is this intentional?